src/server/helpers/source-api-request.ts

```diff
@@ -87,7 +87,7 @@ function enableMockAdapter() {
   });
 }
 
-if (!IS_AP && !process.env.BFF_DISABLE_MOCK_ADAPTER) {
+if (!IS_TAP && process.env.BFF_ENABLE_MOCK_ADAPTER === 'true') {
```

Comment:
Hebben deze string boolean waardes te maken met de azure overgang?

Reply:
Nee. Je zou ook op 'jahoor-doemaar' kunnen testen. Maar `BFF_ENABLE` is explicieter dan `!BFF_DISABLE`